format/spec.md

```diff
@@ -684,7 +684,7 @@ Statistics files metadata within `statistics` table metadata field is a struct w
 
 | v1 | v2 | Field name | Type | Description |
 |----|----|------------|------|-------------|
-| _required_ | _required_ | **`snapshot-id`** | `string` | ID of the Iceberg table's snapshot the statistics were computed from. |
+| _required_ | _required_ | **`snapshot-id`** | `string` | ID of the Iceberg table's snapshot that is using this statistics file. |
```

Comment:
below line number 699, has a `snapshot-id` required field in `Blob metadata` which stores the snapshot-id where the stats are actually computed from. 

So, this field in `StatisticsFile` can be the referred snapshot id. 
So, Incase of rewrite data files, there can be two `StatisticsFile` with two different snapshot id pointing to the same stats file location. This way the expire snapshot can know that the stats file is still referenced.

Reply:
"ID of the Iceberg table's snapshot the statistics file is associated with" ?